src/com/google/javascript/jscomp/parsing/IRFactory.java

```diff
@@ -196,6 +196,10 @@
 
   static final String UNDEFINED_LABEL = "undefined label \"%s\"";
 
+  static final String EXPOSE_DEPRECATED_WARNING =
```

Comment:
Done. It's not suppressible with an `@suppress` annotation. I'm assuming that's because it's a parser warning.